src/app/problem/edit/Description.css

```diff
@@ -0,0 +1,71 @@
+/* TODO: the tabs part (.tabs, .active-tabs, .description-button) */
```

Comment:
1. use `scss` syntax, so that we can easily add nested styling, see [ExamHome.scss](https://github.com/Judge-Girl/Judge-Girl-CMS/blob/master/src/app/exam/ExamHome.scss) as an example
2. Remove unnecessary comments